content/zh-cn/blog/_posts/2024-12-11-api-streaming/index.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: blog
+title: '使用 API 流式传输来增强 Kubernetes API 服务器效率'
+date: 2024-12-11
+slug: kube-apiserver-api-streaming
+author: >
+  Stefan Schimanski (Upbound),
+  Wojciech Tyczynski (Google),
+  Lukasz Szaszkiewicz (Red Hat)
+translator: >
+  [Michael Yao](https://github.com/windsonsea) (DaoCloud)
```

Comment:
The original article is a draft (see its front matter)

/hold

to ensure it doesn't merge early.
Only unhold this when the original article is no longer a draft.